### PR TITLE
Redirect->signin in case of signout in another tab

### PR DESCRIPTION
**Pull request by piotr-travisci**

### CODE REVIEW

File: app/services/auth.js [hunk 1]
```diff
@@ -83,6 +83,29 @@ export default Service.extend({
 
   redirectUrl: null,
 
+  init() {
+    this._super(...arguments);
+    window.addEventListener('focus', this.checkAuth.bind(this));
```
Comment (artursmirnov) [inline app/services/auth.js:88]: This `bind` signature looks a little bit awkward to me :) I'd suggest an arrow function instead:
```suggestion
    window.addEventListener('focus', () => this.checkAuth());
```
Just an optional suggestion though.

File: app/services/auth.js [hunk 2]
```diff
@@ -83,6 +83,29 @@ export default Service.extend({
...
+      return;
+    const { accounts } = this.storage;
+    const { vcsId } = this.currentUser;
+    let stillLoggedIn = false;
+
+    accounts.map((user) => {
+      if (user.vcsId === vcsId) {
+        stillLoggedIn = true;
+      }
+    });
```
Comment (artursmirnov) [inline app/services/auth.js:102]: There's a simpler way to do this with Ember:
```suggestion
    const stillLoggedIn = accounts.isAny('vcsId', vcsId);
```
Here's more info: https://api.emberjs.com/ember/3.12/classes/EmberArray/methods/isAny?anchor=isAny

File: app/services/auth.js [hunk 3]
```diff
@@ -83,6 +83,29 @@ export default Service.extend({
...
+
+    if (!stillLoggedIn) {
+      this.router.transitionTo('signin');
```
Comment (artursmirnov) [inline app/services/auth.js:105]: Are you sure, we don't want to run the full `signOut` with all teardowns and cleanups?

File: app/services/auth.js (hunk 1 above)
Comment (piotr-travisci) [reply app/services/auth.js:88]: You are right. Arrow functions look a way more trendy :-)

File: app/services/auth.js (hunk 2 above)
Comment (piotr-travisci) [reply app/services/auth.js:102]: Awesome, thank you @artursmirnov !

File: app/services/auth.js (hunk 3 above)
Comment (piotr-travisci) [reply app/services/auth.js:99]: This is already performed in another tab - this tab, which signed out before.

**Pull request merged**